src/main/java/com/iexec/common/replicate/ReplicateStatus.java

```diff
@@ -71,14 +78,26 @@ public static boolean isFailedBeforeComputed(ReplicateStatus status) {
         return status.ordinal() < COMPUTED.ordinal() && getFailureStatuses().contains(status);
     }
 
+    /**
+     * @deprecated This unused method will be cleaned.
+     */
+    @Deprecated(forRemoval = true)
     public static boolean isCompletable(ReplicateStatus status) {
```

Comment:
Unnecessary deprecation, can be removed with #407

Reply:
Please see https://github.com/iExecBlockchainComputing/iexec-common/pull/408/commits/2c4b5f8d52afd85cab983baa0b04738efd738c89.

Reply:
Thanks